### ISSUE TITLE
Is it possible to add support for 30XX graphics cards?

### ISSUE THREAD

**Issue by Yang-Maxim** [alievk/avatarify-desktop#27]
Is it possible to add support for 30XX graphics cards?

**Comment by JohanAR** [comment]
I *think* what's needed is the following:
1) Change to higher version libtorch / CUDA
2) Update all dependencies so they're compatible with the new libraries
3) Verify that this doesn't break Avatarify Desktop for other graphics cards

The reason for why this isn't happening on its own is that I believe alievk is busy with the mobile versions of Avatarify, which are higher prioritized since they actually help put food on his table :) I'm a volunteer co-maintainer of the project, but I'm using Linux 
and I haven't even had time to port Avatarify Desktop to Linux yet. In addition neither of us owns a RTX 30XX card, which might slow down the task if either of us were to attempt it.

But since this project is open source, anyone can download the code, fix the issue, and submit a pull request

**Comment by Yang-Maxim** [comment]
Is it okay if I want to train my own model and replace it?

**Comment by JohanAR** [comment]
> Is it okay if I want to train my own model and replace it?

Yes you may. alievk has decided to put a CC0 license on this project, which is very permissive. But if you're just thinking about downloading the source and modifying things for your own use, then you can do pretty much anything. Licenses mostly become relevant if you want to redistribute the changed code.
